src/source/vector_tile_source.js

```diff
@@ -154,7 +154,19 @@ class VectorTileSource extends Evented implements Source {
      * Sets the source `tiles` property and re-renders the map.
      *
      * @param {string[]} tiles An array of one or more tile source URLs, as in the TileJSON spec.
-     * @returns {VectorTileSource} this
+     * @returns {VectorTileSource} Returns itself to allow for method chaining.
+     * @example
+     * map.addSource('vector_source_id', {
+     *     type: 'vector',
+     *     tiles: ['https://some_end_point.net/{z}/{x}/{y}.mvt'],
+     *     minzoom: 6,
+     *     maxzoom: 14
+     * });
+     *
+     * const vectorTileSource = map.getSource('vector_source_id');
+     *
+     * // Update tiles a new endpoint
```

Comment:
@mapbox/gl-js This comment seems to be missing a word, but I'm not sure what. 🤔  "Update tiles to a new endpoint"? "Update tiles on a new endpoint"?

Reply:
What about, "Sets the endpoint associated with a vector tile source."

Reply:
Nitpick, singular, possibly even a colon?

> Set the endpoint associated with a vector tile source:

I think it's referring to the function of the single example below.